src/renderers/shaders/ShaderChunk/lights_physical_fragment.glsl.js

```diff
@@ -21,12 +21,28 @@ material.specularRoughness = min( material.specularRoughness, 1.0 );
 
 #ifdef CLEARCOAT
 
-	material.clearcoat = saturate( clearcoat ); // Burley clearcoat model
-	material.clearcoatRoughness = max( clearcoatRoughness, 0.0525 );
+  material.clearcoat = clearcoat;
+  material.clearcoatRoughness = clearcoatRoughness;
+
+  #ifdef USE_CLEARCOATMAP
+
+    material.clearcoat *= texture2D( clearcoatMap, vUv ).x;
+
+  #endif
+
+  #ifdef USE_CLEARCOAT_ROUGHNESSMAP
+
+    material.clearcoatRoughness *= texture2D( clearcoatRoughnessMap, vUv ).x;
```

Comment:
In the [current gltf extension proposal](https://github.com/KhronosGroup/glTF/pull/1756), the values of clearcoat roughness texture seems sampled from G (Y) channel, not R(X).

Reply:
Ahh good catch, thanks. Let's discuss there, I'm not sure. https://github.com/KhronosGroup/glTF/pull/1756/files#r381750517

Reply:
Updated. ✅